source/includes/oai-pmh/_root.md

```diff
@@ -16,6 +16,52 @@ not available for download.
 
 `https://zenodo.org/oai2d`
 
+
+## First steps with Sickle
```

Comment:
Since we don't necessarily "recommend" Sickle (or Python) for harvesting the OAI-PMH API, I would just include the Python examples in the code blocks (i.e. nothing added in the Markdown "text" left side). 

Any needed text can be written in Python comments.